hocr-eval-lines

```diff
@@ -73,7 +73,7 @@ optlist,args = getopt.getopt(sys.argv[1:],"v")
 
 verbose = (assoc('-v',optlist)=='')
 truth_lines = open(args[0]).read().split('\n')
-actual_doc = html.fromstring(open(args[1]).read())
+actual_doc = html.parse(args[1])
```

Comment:
@kba, do we need the same kind of change for the remaining seven code locations with `html.fromstring`, too?